### DISCUSSION TITLE
Update Ikea devices via ota [failed]

### DISCUSSION THREAD

**Discussion by cavemaster** [Koenkk/zigbee2mqtt#11809] [Devices]
Hello,
I try to update this device [IKEA Button](https://www.zigbee2mqtt.io/devices/E1743.html#ikea-e1743) and get the following error message:

`error Update of 'IKEA_Button_04' failed (Device didn't respond to OTA request)
debug Error: Device didn't respond to OTA request at requestOTA (/app/node_modules/zigbee-herdsman-converters/lib/ota/common.js:196:15) at Object.updateToLatest (/app/node_modules/zigbee-herdsman-converters/lib/ota/common.js:283:21) at OTAUpdate.onMQTTMessage (/app/lib/extension/otaUpdate.ts:256:21)`

I’m using Zigbee2Mqqt in a docker and an USB Sonoff Stick.

Are there any hints what I can do?

Thanks in advance.

Thomas

**Comment by alainkovacs** [comment]
Hi Thomas,
Have you tried pressing the button just before you press "Update device firmware"? First you need to "wake up" the remote, and then it will be able to receive commands. I hope this helps

**Comment by cavemaster** [comment]
It's been a while but the summer is over now.
I still can't update even if I press the button before.
Any other hints I should try?
